lib/rollup/query/geo_totals.ex

```diff
@@ -0,0 +1,63 @@
+defmodule Castle.Rollup.Query.GeoTotals do
+  import Ecto.Query
+
+  def podcast(id, %{from: from, to: to}, %{name: grouping_name}) do
+    podcast(id, from, to, grouping_name)
+  end
+  def podcast(id, from, to, grouping_name, grouping_limit \\ nil) do
```

Comment:
So right now, I put _no_ limit on the number returned by geo-totals.  Which can actually get to be quite a long list, for `geo_subdiv`.  May just have to implement some filtering by country there.

Reply:
I was imagining a drilldown into subdivs by country, at least as far as I've worked with the geochart library and the geo json datasets I've seen.